frontend/src/services/auth.service.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+const API_URL = "http://localhost:5000/auth/";
```

Comment:
Some documentation of how this authorization works would be helpful

Reply:
<img width="780" alt="user signup process" src="https://user-images.githubusercontent.com/63368928/204991080-6a2f0d99-45f4-4044-a5b0-cee920f721a0.png">
This is the user signup process.

Reply:
Yeah, this graphic helps a lot